### PR TITLE
F/unscheduled classes box

### PR DESCRIPTION
**Pull request by albirdy98**
Made changes to unscheduled-classes-box, edited MainView

### CODE REVIEW

**Comment by Joonpark13** [comment]
Looks like you guys have conflicts, would be great if you could rebase and resolve those before I review again. Also since this contains a visual change, can we get a screenshot put up?

File: src/components/App.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ class App extends Component {
         <TopBar />
         <Grid container>
           <Grid item md={9} sm={12} xs={12} className={classes.calendar}>
-            <CalendarViewContainer />
+            <MainView />
```
Comment (Joonpark13) [inline src/components/App.js]: Let's rename this to `ClassesView` - I like that you realized `CalendarView` is no longer accurate, but specificity is always better than not!

**Comment by kevinjihwanlee** [comment]
[Unscheduled Classes Box](https://trello.com/c/GWtpAo6u/48-unscheduled-classes-box)

**Comment by albirdy98** [comment]
Added a Unscheduled Classes tab next to Calendar tab
screen shot 2019-02-20 at 10 33 49 pm
screen shot 2019-02-20 at 10 41 02 pm

**Comment by Joonpark13** [comment]
screen shot 2019-02-21 at 7 33 17 pm
I'm seeing the tabs go full width - let's try to make them more like your screenshot. Also make sure the heading is "Unscheduled Classes", not "courses", to match with what's in the tab. Lastly, I don't think you need the paper - let's just throw the typography in there as the heading for now and we'll go back and style it later.

File: src/components/App.js [hunk 2]
```diff
@@ -41,11 +42,14 @@ class App extends Component {
...
         <Grid container>
           <Grid item md={9} sm={12} xs={12} className={classes.calendar}>
-            <CalendarView />
+            <ClassesView />
           </Grid>
           <Grid item md={3} sm={12} xs={12}>
             <SidebarView />
           </Grid>
+          <Grid item md={9} sm={12} xs={12}>
+            <UnscheduledClasses />
```
Comment (Joonpark13) [inline src/components/App.js]: I think you forgot to take this out after putting in the tab.

**Review by Joonpark13 — CHANGES_REQUESTED** [review]

File: src/components/App.js (hunk 2 above)
Comment (kevinjihwanlee) [reply src/components/App.js]: oops that was sloppy - fixed

**Review by Joonpark13 — APPROVED** [review]

**Pull request merged**